homeassistant/components/renson/sensor.py

```diff
@@ -172,14 +172,12 @@ class RensonSensorEntityDescription(
         raw_format=False,
         device_class=SensorDeviceClass.TEMPERATURE,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
-        entity_registry_enabled_default=False,
     ),
     RensonSensorEntityDescription(
         key="BREEZE_LEVEL_FIELD",
         translation_key="breeze_level",
         field=BREEZE_LEVEL_FIELD,
         raw_format=False,
-        entity_registry_enabled_default=False,
```

Comment:
Why is this sensor no longer disabled, isn't it enough to show the level as the fan speed?

Reply:
Breeze level is a setting inside the device and can be set using the official application. When toggling the breeze switch it will take this value as fan speed.

Reply:
Yes, i saw it's used in the fan entity to set and get fan speed when breeze mode is enabled. Why is an additional sensor useful?

Reply:
More info about the device that users can view.

Reply:
OK